roles/repos/tasks/apt.yml

```diff
@@ -11,6 +11,7 @@
   register: result
   until: result|succeeded
   retries: 5
+  tags: apt
```

Comment:
In my experience, the more tags the better
Can we do the general 'apt' tag (like you've already done)
But add a specific one for each action.
So for this, the tags could be: `apt-keys`, `apt`